src/components/application_manager/test/policy_handler_test.cc

```diff
@@ -1387,6 +1387,21 @@ TEST_F(PolicyHandlerTest, IsRequestTypeAllowed) {
   policy_handler_.IsRequestTypeAllowed(kPolicyAppId_, type);
 }
 
+TEST_F(PolicyHandlerTest, IsRequestSubTypeAllowed) {
+  // Arrange
+  EnablePolicyAndPolicyManagerMock();
+
+  // Check expectations
+  EXPECT_CALL(*mock_policy_manager_, GetAppRequestSubTypesState(kPolicyAppId_))
+      .WillOnce(Return(policy::RequestSubType::State::AVAILABLE));
+  EXPECT_CALL(*mock_policy_manager_, GetAppRequestSubTypes(kPolicyAppId_))
+      .WillOnce(Return(std::vector<std::string>({"OEM_SPECIFIC", "HTTP"})));
+
+  // Act
+  const std::string& subtype = "OEM_SPECIFIC";
```

Comment:
@BSolonenko Why reference?

Reply:
@AByzhynar Fixed in https://github.com/smartdevicelink/sdl_core/pull/2151/commits/2c2298ee5f7f78c65b0e9f7c23f06fa49da86344